main.py

```diff
@@ -7,6 +7,7 @@
 def print_hi(name):
     # Use a breakpoint in the code line below to debug your script.
     print(f'Hi, {name}')  # Press Ctrl+F8 to toggle the breakpoint.
+    print("This is new")
```

Comment:
This looks good but perhaps change this
```suggestions
Nandan changes!